### PR TITLE
EMSUSD-278 - Embed import options in subheadings 

### PR DESCRIPTION
**Pull request by samuelliu-adsk**
- Embed import options in subheading
- Change the style from progressive disclosure to Maya style (grey out children options when parent unselected)

### CODE REVIEW

**Comment by samuelliu-adsk** [comment]
Preview of the change:
image

File: plugin/adsk/scripts/mayaUsdTranslatorImport.mel [hunk 1]
```diff
@@ -229,24 +229,26 @@ global proc int mayaUsdTranslatorImport (string $parent,
 //            menuItem -label "Object space";
 //            menuItem -label "World space";
 //        optionMenuGrp -e -sl 1 mayaUsdTranslator_CoordSystemOptionMenu;
+        frameLayout -label `getMayaUsdString("kImportMaterialsLbl")` materialsFrameLayout;
+            separator -style "none";
+            checkBoxGrp -label "" -label1 `getMayaUsdString("kImportMaterialsLbl")` -cw 1 $cw1 -value1 1 -ann `getMayaUsdString("kImportMaterialsAnn")` -cc ("mayaUsdTranslatorImport_MaterialsCB") mayaUsdTranslator_MaterialsCheckBox;
```
Comment (seando-adsk) [inline plugin/adsk/scripts/mayaUsdTranslatorImport.mel:234]: I'm not sure what the default is, but I think you should pass the number of checkboxes with `-ncb 1` and I don't think you need `-label ""` as that flag is optional. Same with all the ones below.

**Review by seando-adsk — COMMENTED** [review]
Just one minor comment.

File: plugin/adsk/scripts/mayaUsdTranslatorImport.mel (hunk 1 above)
Comment (samuelliu-adsk) [reply plugin/adsk/scripts/mayaUsdTranslatorImport.mel:234]: Hi @seando-adsk , `-label ""` is actually needed to achieve the alignment. Without `-label` the option will be aligned to the left as shown below: 
image

For `-ncb` I'm not sure it's necessary.. We didn't use that option before so I think that's more for the case that we have multiple checkboxes?

Comment (seando-adsk) [reply plugin/adsk/scripts/mayaUsdTranslatorImport.mel:234]: Okay for the empty label - I didn't realize that.
I checked the code for the checkBoxGrp command and it does indeed default to 1.

**Review by seando-adsk — APPROVED** [review]

**Pull request merged**